CHANGELOG.md

```diff
@@ -22,7 +32,7 @@
 
 - mark compatible with Taskgraph 6.x
 
-## 1.0.0 (2023-12-20)
+## 1.0.0 (2023-07-28)
```

Comment:
This is because I had initially forgotten to tag this release, then tagged it after the fact to generate the changelog. Then I realized the date comes from the date of the tag and not the date of the commit, and re-did the tag with the proper date but didn't re-do the changelog.